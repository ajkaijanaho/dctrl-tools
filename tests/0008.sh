#!/bin/sh

set -e

$JOIN_DCTRL -j Package -o 0,1.Version:Old-Version,2.Version:New-Version -a1 0007.stat 0007.pkg 