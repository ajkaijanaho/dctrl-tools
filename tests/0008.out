Package: aalib1
Old-Version: 1.4p5-22

Package: abiword
Old-Version: 2.4.6-1.1
New-Version: 2.4.6-1.1

Package: abiword-common
Old-Version: 2.4.6-1.1
New-Version: 2.4.6-1.1

Package: abiword-help
Old-Version: 2.4.6-1.1
New-Version: 2.4.6-1.1

Package: abiword-plugins
Old-Version: 2.4.6-0
New-Version: 2.4.6-1.1

Package: abiword-plugins-gnome
Old-Version: 
New-Version: 2.4.6-1.1

